src/stories/pagefold/Pagefold.tsx

```diff
@@ -25,15 +25,13 @@ export const Pagefold = (props: PagefoldProps) => {
   const classes = {
     wrapper: clsx(
       `pagefold-parent--${size}`,
-      {"internal-pagefold-parent": isAContainer},
+      { "internal-pagefold-parent": isAContainer },
       className
     ),
-    content: clsx(
-      type,
-      `pagefold-triangle--${size}`,
-      {"pagefold-inherit-parent": isInheriting}
-    )
-  }
+    content: clsx(type, `pagefold-triangle--${size}`, {
```

Comment:
Honestly I find the former syntax more readable. But don't want to be square about it.